src/tests/universal-nrp-e2e/StartTests.sh

```diff
@@ -3,11 +3,15 @@
 # StartTests.sh
 # Description: This Microsoft internal only script is used to download the latest Azure Policy packages from internal pipelines
 #              and run the tests on the specified test data provided.
-# Usage: ./StartTests.sh [-r run-id] [-m vm-memory-mb] [-j max-concurrent-jobs]
+# Usage: ./StartTests.sh [-r run-id] [-m vm-memory-mb] [-j max-concurrent-jobs] [-d policy-package-directory] [-n no-gui]
+# Options:
 #        -r run-id: Specify the run-id of the pipeline run to download the packages from (Default: latest-succeeded-run-id)
 #        -m vm-memory-mb: Specify the memory in MB to be used for the VMs (Default: 512)
 #        -j max-concurrent-jobs: Specify the maximum number of concurrent jobs to run the tests (Default: 5)
-# Dependencies: cloud-localds, qemu-system-x86, jq, az
+#        -d policy-package-directory: Specify the directory containing the policy packages to use (Default: download from Azure DevOps pipeline)
+#        -n no-gui: Disable GUI for the VMs (Default: false)
```

Comment:
>n no-gui: Disable GUI for the VMs (Default: false) [](http://example.com/codeflow?start=10&length=50)

Why is default not true for disabling GUI?

Reply:
we want the GUI in general as its easier to debug but it wont work in headless environments so i added this flag to explicitly disable it